exareme2/controller/controller.py

```diff
@@ -68,9 +69,12 @@ class NodesTasksHandlers:
 class InitializationParams:
     smpc_enabled: bool
     smpc_optional: bool
+
     celery_tasks_timeout: int
     celery_run_udf_task_timeout: int
 
+    dp_params: Optional[DifferentialPrivacyParams] = None
```

Comment:
A potential problem that I see here and in many other places is that the smpc params are separate from the dp params. 

DP cannot exist without SMPC so we need either to protect from this ` if dp_params and not smpc_enable: raise Error(...)`
or to contain both the smpc params together with the dp params as a subset.

This should probably also be the case in the controller's `config.toml` and in the `README.md` where `dp` is completely separate than the smpc but it cannot actually exist separately.

Reply:
fixed